libs/amp-evaluation/.env.example

```
@@ -7,272 +7,46 @@
-
-# Cron expression (e.g., "0 */6 * * *" = every 6 hours)
-SCHEDULER_CRON_EXPRESSION=0 */6 * * *
+# Set AMP_TRACE_API_URL to fetch live traces from the platform, OR
+# set AMP_TRACE_FILE_PATH to load traces from a local JSON file.
 
-# Trace lookback window (how far back to fetch traces)
-# Format: ISO 8601 duration (e.g., PT24H = 24 hours, P7D = 7 days)
-TRACE_LOOKBACK_WINDOW=PT24H
+# Option 1: Live traces from platform API
+AMP_TRACE_API_URL=https://platform.example.com
+AMP_TRACE_API_KEY=your_api_key_here  # optional
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AMP_TRACE_API_KEY key should go before the AMP_TRACE_API_URL key

(UnorderedKey)